(1) Normalize attachment filename as JIRA seems to respond with server error on non-ASCII names and (2) Python 3 compatibility fixes

mrts opened the pull request:
See https://answers.atlassian.com/questions/143915/answers/32979258/comments/46688444:

> The only issue I have is while adding the attatchment to the new Jira ticket. I always get an error 500 with an empty error text. Do you have any idea what could be the cause here?
> I have identified the issue: The script currently doesn't support special characters like ä or é for attachment file names.

captain1701e commented on lib/export_import.py:
```diff
@@ -115,9 +116,13 @@ def _add_attachments(issue, jira, attachments):
                     attachment=buf)
 
+def _normalize_filename(value):
+    return unicodedata.normalize('NFKD', value).encode('ascii', 'ignore')
```
When testing I had to add '.decode("utf-8")' at the end otherwise in client.py the creation of MultipartEncoder would fail because of expected str but given byte.

mrts replied:
Hm, interesting - you are using Python 3? I never tried the script with Python 3, but glad it works :). I'll fix this soon and let you know.

mrts replied:
I pushed the fix, please test. If all is good, I'll merge and we can happily live ever after :)!

mrts commented:
Thanks for the feedback! I fixed the `sorted_globals.sort()` issue and added more Python 3 fixes, it should be good now. Much appreciated if you can test!

mrts commented:
I'll merge as this *look* pretty safe. Please test *master* if you can and even if you cannot, your help so far is much appreciated :)! It will surely help others as well.

captain1701e commented:
I tested master and it worked perfect :+1:

mrts commented:
Glad to hear, thanks a lot for your help 🙂!
